src/storage/limits.ts

```diff
@@ -59,9 +59,12 @@ export function isValidKey(key: string): boolean {
 export function isValidBucketName(bucketName: string): boolean {
   // only allow s3 safe characters and characters which require special handling for now
   // https://docs.aws.amazon.com/AmazonS3/latest/userguide/object-keys.html
+  // https://docs.aws.amazon.com/AmazonS3/latest/userguide/bucketnamingrules.html
```

Comment:
Maybe add a tiny note here that we just take the length (and slash) restriction from bucket and the rest of the rules are based on S3 object validation.